tests/mantapay/rpc_correctness_test.ts

```diff
@@ -0,0 +1,104 @@
+import { ApiPromise, WsProvider } from "@polkadot/api";
```

Comment:
If this file's functionality is replaced, why not just delete it.

Reply:
Done.